### DISCUSSION TITLE
Best way to train two-speaker embeddings

### DISCUSSION THREAD

**Discussion by picheny-nyu** [pyannote/pyannote-audio#796] [Q&A]
I have an application in which I know a priori there are only two speakers, but the conditions are low snr and not much data, maybe an hour or so of total speech. Any suggestions for what sort of configuration to use to train embeddings? 

Thanks
Michael

**Comment by hbredin** [comment]
I would fine-tune [this segmentation model](https://huggingface.co/pyannote/segmentation).
Note that this relies on upcoming pyannote.audio 2.0 currently available in `develop` branch.
